src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -5721,10 +5721,9 @@ class A                                                               \
 
             Assert.False(result.ContainsErrors, $"Compilation error(s) occurred: {result.Output} {result.Errors}");
 
-            var assembly = Assembly.LoadFrom(Path.Combine(tempDir, name));
-            assembly.ManifestModule.GetPEKind(out PortableExecutableKinds peKind, out _);
-
-            Assert.Equal(PortableExecutableKinds.ILOnly, peKind);
+            var assemblyName = AssemblyName.GetAssemblyName(Path.Combine(tempDir, name));
+            Assert.Equal(name.Replace(".dll", ", Version=0.0.0.0, Culture=neutral, PublicKeyToken=null"),
```

Comment:
Instead of testing the name, why don't you use hello world as the input source and then run the emitted exe? That's basically a full end-to-end test.

Reply:
This test is for compiling library (with empty method) from stdin source; the one above does the exe E2E testing.

Reply:
Ah, I see.